lti_emailer/canvas_api_client.py

```diff
@@ -59,3 +59,24 @@ def get_enrollments(canvas_course_id, section_id):
             raise
         cache.set(cache_key, result)
     return result
+
+
+def get_teacher_enrollments(canvas_course_id):
```

Comment:
Can we use the existing cached enrollments here and filter out the non-"teacher" ones?